.Rbuildignore

```diff
@@ -1,2 +1,3 @@
 ^tempConverter\.Rproj$
 ^\.Rproj\.user$
+.github
```

Comment:
It's recommended to have a new line at the end of the file (that's why Git and GitHub usually warn about this). Otherwise, if you add line 4 after line 3, you also have to modify line 3 adding a whitespace character, although the line 3 itself needs not to be changed.